assets/stylesheets/bootstrap/_theme.scss

```diff
@@ -48,6 +49,7 @@
 // Mixin for generating new styles
 @mixin btn-styles($btn-color: #555) {
   @include gradient-vertical($start-color: $btn-color, $end-color: darken($btn-color, 12%));
+  @include reset-filter; // Disable gradients for IE9 because filter bleeds through rounded corners; see https://github.com/twbs/bootstrap/issues/10620
```

Comment:
I'm not sure what reset-filter is doing here, are you sure we need this?  At the very least remove the mention of IE9 if you're 100% sure that this is somehow required (doubtful).